src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball(); // color and number are created in the default constructor
+        return ball;
```

Comment:
this defeats the purpose of the task, Lottery class should be responsible for generating a ball, what if we want to create a plain ball, without generating random color and number? Your implementation doesn't allow this